content/03.methods.md

```diff
@@ -185,3 +193,4 @@ We retained articles that were included in the PMCOA corpus and performed a toke
 As above, we generated document embeddings for every obtained preprint and published article.
 We projected these preprint embeddings onto our publication landscape to visually observe the dispersion of this subset.
 Finally, we performed a time analysis that paralleled our approach for the full set of preprint-publication pairs to examine relationships between linguistic changes and the time to publication.   
+We also performed a supplemental analysis where we filtered out preprint-publication pairs within our bioRxiv snapshot that had a publication time longer than the slowest publication pair in [@doi:10.1101/2021.02.20.432090].
```

Comment:
This is not enough context! What was the concern that you were trying to control for? Help the reader get into a head space where they can figure out what this means 😁 
```suggestion
We also performed a secondary analysis where we filtered out preprint-publication pairs within our bioRxiv snapshot that had a publication time longer than the slowest publication pair in [@doi:10.1101/2021.02.20.432090].
```